CHANGELOG.md

```diff
@@ -0,0 +1,53 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
Are we strictly-speaking following semver at this point?  I thought the plan was to commit starting from 1.0.0, or maybe from 0.1.0 -- recent "patch" versions certainly haven't been backwards compatible.